webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js

```diff
@@ -36,31 +52,15 @@ const ContentViewVersionRepositoryCell = ({
     },
     docker_tag: {
       name: __('Container image tags'),
```

Comment:
The docker_manifest and docker_manifest:_list URLs have an extra /content in the URL.

Reply:
Oh.. Strange. Good catch!